examples/demo_integrate_limits.py

```diff
@@ -0,0 +1,78 @@
+"""
+Demonstrate Integrate Limits
+============================
+
+For this demonstration, we generate inversion
```

Comment:
this doesn't run for me -- see below:
```
$ py demo_integrate_limits.py
logging output to C:\Users\johnf\pyspecdata.0.log
C:\Users\johnf\pyspecdata.0.log appears to be locked or otherwise inaccessible:
I'm going to explore other options for fileno
logging output to C:\Users\johnf\pyspecdata.1.log
--> integrate_limits.py(70):root     integrate_limits 2021-08-24 09:23:08,218
INFO: FILTER WIDTH IS 0.0043043044
--> integrate_limits.py(70):root     integrate_limits 2021-08-24 09:23:08,218
INFO: FILTER WIDTH IS 0.0043043044
Determined frequency limits via Lorentzian filter of [  0.         194.23828125]

Traceback (most recent call last):
  File "demo_integrate_limits.py", line 68, in <module>
    freq_lim = integrate_limits(data['ph1',0]['ph2',1],
TypeError: integrate_limits() got an unexpected keyword argument 'Gaussian'
(base)
johnf@DESKTOP-O82GMEV MINGW64 ~/git_repos/proc_scripts/examples (fix_integrate_limits)
$ git rev-parse @
b9d5e4530ac19c32b8b7113ca6a7f3b52949a151
(base)
johnf@DESKTOP-O82GMEV MINGW64 ~/git_repos/proc_scripts/examples (fix_integrate_limits)
$ cd ~/notebook/pyspecdata/
(base)
johnf@DESKTOP-O82GMEV MINGW64 ~/notebook/pyspecdata (master)
$ git rev-parse @
cc24092916c5b97a3b1656f5b04b8ad14c5cc6dc
```

Reply:
(the `git rev-parse` lines are to show you the commits that I'm using)

Reply:
I see - I only made the necessary change to the real data script, not the fake data script. I will add it now.

Reply:
New version has been pushed and should work now for you.